packages/edition-twig/patternlab-config.json

```diff
@@ -0,0 +1,144 @@
+{
+  "engines": {
+    "twig": {
+      "namespaces": [
```

Comment:
@EvanLovely three big questions here:

1. Are these recursive by default?
2. How do we configure PHP Twig extensions?
3. For that matter, how would we configure any of the other Twig Renderer config options: 
https://github.com/basaltinc/twig-renderer/blob/master/config.schema.json

Reply:
1. Not by default, but that's just a `recursive: true` away. I want to update the Twig StarterKit demo to change things like `@atoms/04-folder/01-item.twig` to `@atoms/01-item.twig` as well.
2. Just set `engines.twig.alterTwigEnv` - you can see where the config is [passed in here](https://github.com/pattern-lab/patternlab-node/blob/004fd597de0338b03e4da86bc423503054bba431/packages/engine-twig-php/lib/engine_twig_php.js#L58)
3. I want to review the config a bit more on TwigRenderer before I lock it down (and release 1.0.0). Then we just pass more config in like the above link.